qemu/tests/hugepage_specify_node.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+from virttest.staging import utils_memory
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu allocate hugepage from node.
+    Steps:
+    1) System setup hugepage on host.
+    2) Mount this hugepage to /mnt/kvm_hugepage.
+    3) Boot guest only allocate hugepage from node.
+    4) Check the hugepage used only from node.
+    :params test: QEMU test object.
+    :params params: Dictionary with the test parameters.
+    :params env: Dictionary with test environment.
+    """
+    error_context.context("Get host numa topological structure.", logging.info)
+    host_numa_node = utils_misc.NumaInfo()
+    node_list = host_numa_node.online_nodes
+
+    pre_hugepages_Free = {}
```

Comment:
mixed case in var name is not a good idea.

Reply:
Thinks a lot, correcting.